### PR TITLE
fix image issue

### PR DESCRIPTION
**Pull request by aliceriot**
#### Pre-Flight checklist

- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?

no ticket

#### What's this PR do?

fixes an issue where the default course image wasn't working when deployed (it was, strangely, working locally...)

#### How should this be manually tested?

course images should all be working, whether it's one specific to a course or the default one.

### CODE REVIEW

File: site/config.toml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-baseurl = ""
+baseurl = "ocw-next.netlify.com"
```
Comment (gumaerc) [inline site/config.toml]: This looks great locally.  On the deploy preview if you hit the placeholder landing page, you'll notice that the link to courses is rendering as https://deploy-preview-99--ocw-next.netlify.com/ocw-next.netlify.com/courses/.  Looking at the placeholder index, the link is like this:

```
<a href="./courses/">
  course listing
</a>
```

I think if we set the base URL, we're going to have to use the absURL or relURL shortcodes contextually for any link in any template contextually.

Comment (aliceriot) [reply site/config.toml]: ugh what a mess :woman_facepalming:

Comment (gumaerc) [reply site/config.toml]: Yea, luckily crawling through the code I think that's actually the only raw relative URL hard coded in our templates, so you should be good to just change that one.

File: netlify.toml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[context.deploy-preview]
+command = "hugo -b $DEPLOY_PRIME_URL"
+
+[context.deploy-preview.environment]
+HUGO_VERSION = "0.60"
```
Comment (gumaerc) [inline netlify.toml]: I would remove the specific version tag here, as we haven't pinned a version yet and are just using latest at the moment.  Either that or we can pin the version of `hugo-bin` in our `package.json`.

Comment (aliceriot) [reply netlify.toml]: it's pinned in our `yarn.lock` to that version

Comment (aliceriot) [reply netlify.toml]: buuut netlify doesn't seem to like that so I will remove

**Comment by aliceriot** [comment]
just realized that I think we need to just change the command in `package.json`, so trying that out

**Comment by gumaerc** [comment]
[Course image pathing issue](https://trello.com/c/j7ai6fEC/58-course-image-pathing-issue)

**Comment by gumaerc** [comment]
:+1: APPROVED

**Pull request merged**